# each Fusion scan point info
uint8    device_id
uint8    type
uint16   number
float32  start_angle
float32  end_angle
uint64   scan_start_time
uint64   scan_end_time
uint64   device_start_time
uint64   device_end_time
float32  scan_frequency     # Hz
float32  beam_tilt
uint32   scan_flags
MountPosition mount_position
Resolution  resolution1
Resolution  resolution2
Resolution  resolution3
Resolution  resolution4
Resolution  resolution5
Resolution  resolution6
Resolution  resolution7
Resolution  resolution8
